evalai/submissions.py

```diff
@@ -140,3 +154,45 @@ def push(image, phase):
                     if k != "progressDetail"
                 )
             )
+
+
+@click.command()
+@click.argument("URL", nargs=1)
+def download_file(url):
+    parsed_url = urlparse.urlparse(url)
+    bucket = urlparse.parse_qs(parsed_url.query).get("bucket")
+    key = urlparse.parse_qs(parsed_url.query).get("key")
+    request_path = URLS.download_file.value
+    request_path = request_path.format(bucket[0], key[0])
+    response = make_request(request_path, "GET")
+    signed_url = response.get("signed_url")
+    file_name = key[0].split("/")[-1]
+    try:
+        response = requests.get(signed_url, stream=True)
+        with open(file_name, "wb") as file:
+            total_length = int(response.headers.get("content-length"))
+            for ch in progress.bar(
```

Comment:
Clint library is archived. See https://github.com/kennethreitz/clint

Please use some other progressbar library such as: https://pypi.org/project/progress/